planner_cspace/src/planner_3d.cpp

```diff
@@ -1136,6 +1276,7 @@ class Planner3dNode
     srs_forget_ = neonavigation_common::compat::advertiseService(
         nh_, "forget_planning_cost",
         pnh_, "forget", &Planner3dNode::cbForget, this);
+    srs_make_plan_ = pnh_.advertiseService("make_plan", &Planner3dNode::cbMakePlan, this);
```

Comment:
(It looks better to use global namespace. But move_base also provide this service in private ns.)